### PR TITLE
🐛 Appends toasts to the story instead of the body

### LINKED ISSUE #15317: Append story toast messages in the story instead of in the body.
We want to start supporting having multiple stories in the same document.

Right now, toast messages are [appended in the body](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/toast.js#L46). They would be positioned against the body, when we want them to stay inside the story.

Appending them in the ``amp-story`` element should fix this future positioning issue.

### PR DESCRIPTION
**Pull request by addieachan**
- Fixes #15317

### CODE REVIEW

**Comment by addieachan** [comment]
@gmajoulet @newmuis @choumx PTAL

**Comment by addieachan** [comment]
Ahh ok! Changing the show method was my initial approach since Gabriel explained the problem really well, but I think I just lost scope it when I was trying to make things effience. My bad!

**Comment by addieachan** [comment]
Also, do I need to make the change in the 0.1 version as well?

**Comment by newmuis** [comment]
Nope, it should be fine to change this only within 1.0.

File: extensions/amp-story/1.0/toast.js [hunk 1]
```diff
@@ -36,8 +36,9 @@ export class Toast {
   /**
    * @param {!Window} win
    * @param {!Node|string} childNodeOrText
+   * @param {Element} storyEl
    */
-  static show(win, childNodeOrText) {
+  static show(win, childNodeOrText, storyEl) {
```
Comment (newmuis) [inline extensions/amp-story/1.0/toast.js]: You can actually get the window from an element, by doing `const win = toWin(rootElement.ownerDocument.defaultView)`.  Then you won't need to take `win` as a param at all.

As a nit, for some reason I also prefer the element being before the string.

File: extensions/amp-story/1.0/toast.js [hunk 2]
```diff
@@ -34,10 +35,12 @@ const TOAST_VISIBLE_TIME_MS = 2600;
  */
 export class Toast {
   /**
-   * @param {!Window} win
+   * @param {Element} storyEl
```
Comment (newmuis) [inline extensions/amp-story/1.0/toast.js]: nit: `{!Element}`

**Pull request merged**